### PR TITLE
Refs 183: Add ouiaId for pkgcount tooltip and repo actions

### PR DESCRIPTION
**Pull request by mshriver**
might need ones for edit/delete buttons, but they're in a dropdown and should be selectable by name with an ID on the dropdown

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-183

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

File: src/components/ContentListTable/components/PackageCount.tsx [hunk 1]
```diff
@@ -19,8 +19,11 @@ const PackageCount = ({ count, status }: Props) => {
 
   if (!count && status === 'Pending') {
     return (
-      <Tooltip isContentLeftAligned content='Repository has not been introspected yet'>
-        <Text className={classes.text}>N/A</Text>
+      <Tooltip 
+        isContentLeftAligned
+        content=<Text ouiaId='package_count_tooltip' className={classes.text}>Repository has not been introspected yet</Text>
```
Comment (Andrewgdewar) [inline src/components/ContentListTable/components/PackageCount.tsx:24]:
```suggestion
        content={<Text ouiaId='package_count_tooltip' className={classes.text}>Repository has not been introspected yet</Text>}
```

Comment (mshriver) [reply src/components/ContentListTable/components/PackageCount.tsx]: thanks!

**Comment by mshriver** [comment]
After playing with this a bit, I'm going to close this as-is. I'll tackle tooltip reading separately, but the ouia for repo actions is impossible right now in PF4 (ActionsColumn), and the ouiaIDs for pkgcount and status aren't really necessary.

**Pull request closed without merging**